app/controllers/v2/widgets_controller.rb

```diff
@@ -130,9 +130,7 @@ def international_status
   def ecosystem_service
     if params.has_key?(:location_id) && params[:location_id] != "worldwide"
       @location_id = params[:location_id]
-      @data = EcosystemService.select("indicator, location_id, value, sum(value) over () as total_value").where(
```

Comment:
`total_value` was not used at json builder at all in the end, so I am removing it ;)